test/test_redis_asyncio.py

```diff
@@ -293,3 +293,16 @@ async def test_async():
     x = await cache.get("fakeredis")
     # assert
     assert x == b"plz"
+
+
+@pytest.mark.parametrize('nowait', [False, True])
```

Comment:
Check out the readme instructions, you can run your test on `redis-py` v4.4 and above using the annotation:
`@testtools.run_test_if_redispy_ver('above', '4.4')`.
If possible, also add a test with `@testtools.run_test_if_redispy_ver('below', '4.3.4')` where it checks that an error is thrown.

Reply:
redis-py 4.4.0 is not released yet, so compare with rc version.
https://github.com/cunla/fakeredis-py/pull/76/files/7cb5481cbcb3c7b198d1729ed62f88488df11bd0..570a29a3906b0129defdc547fa98d9240f4119fb